Dockerfile.assisted-service

```diff
@@ -35,7 +35,33 @@ RUN cd build && python3 setup.py sdist --dist-dir /assisted-service-client
 FROM quay.io/centos/centos:centos8
 
 # openshift-install requires this
-RUN yum install -y libvirt-libs && yum clean all
+RUN dnf install -y libvirt-libs git python3 && \
+    dnf install -b -y  dnf-plugins-core && \
+    dnf copr enable -y networkmanager/NetworkManager-master && \
+    dnf copr enable -y nmstate/ovs-el8 && \
+    dnf copr enable -y nmstate/nmstate-0.3&& \
+    dnf copr enable -y nmstate/nispor && \
+    dnf install -y pkg-config && \
+    dnf install -y glib2-devel && \
+    dnf install -y gobject-introspection-devel && \
+    dnf install -y cairo-devel && \
+    dnf install -y cairo-gobject-devel && \
+    dnf install -y python3-devel && \
+    dnf install -y nispor && \
+    dnf install -y NetworkManager && \
+    dnf install -y NetworkManager-ovs && \
+    dnf install -y NetworkManager-team && \
+    dnf install -y NetworkManager-config-server && \
+    dnf install -y openvswitch2.11 && \
+    dnf install -y python3-openvswitch2.11 && \
+    pip3 install --user pycairo && \
+    dnf remove -y dnf-plugins-core && \
+    dnf clean all
+
+RUN git clone https://github.com/nmstate/nmstate
```

Comment:
https://github.com/nmstate/nmstate/releases/tag/v1.0.2 was released yesterday.
It can be downloaded from https://github.com/nmstate/nmstate/releases/download/v1.0.2/nmstate-1.0.2.tar.gz, extracted and run installed.

It will spare the git rpm and will allow using a fixated version for now (unless we need to rely on master)

Reply:
Done